server/seed/index.js

```diff
@@ -1,14 +1,22 @@
 var faker = require('faker');
-const models = require('./models/index.js');
-//usernames
-var randomUsername = faker.internet.userName();
-// console.log('randomUsername', randomUsername);//Elton_Hauck
-//product names
-var randomProductName = faker.commerce.productName();
-// console.log('randomProductName', randomProductName);//Practical Soft Towels
-//url images of those product names
-var randomImageUrl = faker.image.fashion();
-// console.log('randomImageUrl', randomImageUrl);//http://lorempixel.com/640/480/fashion
-
-
-var productArray = [];
+const models = require('../models/index.js');
+
+let productSeed = () => {
+  for (let i = 0; i < 10; i++) {
```

Comment:
You might want to generate more than 10 database entries, otherwise looks good

Reply:
Yea definitely. I was just making sure that it works.